package.json

```diff
@@ -41,6 +41,7 @@
     "eslint-plugin-react": "^4.2.3",
     "joi": "^10.6",
     "kafka-node": "^2.2.0",
```

Comment:
Can remove this and update the reference of kafka-node to node-rdkafka in DESIGN.md.

Reply:
thanks, updated